packages/components/src/inputs/src/TextArea.tsx

```diff
@@ -244,7 +244,13 @@ function TextArea(props: TextAreaProps, ref: ForwardedRef<HTMLDivElement>) {
             >
                 <RACTextArea ref={mergedTextAreaRef} placeholder={placeholder} cols={cols} rows={rows} />
 
-                {showCharacterCount && maxLength && <CharacterCount charactersLeft={maxLength - characterCount} isInvalid={overMaxLength} />}
+                {showCharacterCount && maxLength &&
+                    <CharacterCount
+                        charactersLeft={maxLength - characterCount}
```

Comment:
I know it was already named this, but I'm thinking a better name would be "remainingCharacters" since left can have multiple meanings. It may be more clear.